lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -694,6 +694,21 @@ function dosomething_signup_get_mbp_params($account, $node, $opt_in, $options =
     'campaign_country' => $campaign_country_code,
   ];
 
+  // Northstar id.
+  if (module_exists('dosomething_northstar')) {
```

Comment:
can we DRY this and [this](https://github.com/DoSomething/phoenix/pull/7007/files#diff-ad30a5d569ca2ae6d2e04223aa501104R644) up?